std/algorithm/sorting.d

```diff
@@ -2185,14 +2185,17 @@ auto topN(alias less = "a < b",
         Range)(Range r, size_t nth)
     if (isRandomAccessRange!(Range) && hasLength!Range)
 {
-    import std.algorithm : swap; // FIXME
-    import std.random : uniform;
-
     static assert(ss == SwapStrategy.unstable,
             "Stable topN not yet implemented");
+
+    if (nth >= r.length) return r[0 .. $];
+
+    import std.algorithm : swap; // FIXME
```

Comment:
Why "FIXME"? Isn't this how local imports are supposed to work?

Oh I see, it's inherited from older code... looks like the intention was to replace it with `import std.algorithm.mutation : swap;`.